### PR TITLE
docs-first rough concept for 3dsmax host integration

### PR DESCRIPTION
**Pull request by LiborBatek**
## Brief description
First kick out of the 3dsmax integration docs...

### CODE REVIEW

**Review by georgybarklaya — APPROVED** [review]
First version seems to be ok.

File: website/docs/artist_hosts_3dsmax.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+Blender settings concerning framerate, resolution and frame range are handled
+by OpenPype. If set correctly in Ftrack, Blender will automatically set the 
+values for you.
+
+
+## Publishing models
+
+### Intro
+
+Publishing models in Blender is pretty straightforward. Create your model as you
```
Comment (BigRoy) [inline website/docs/artist_hosts_3dsmax.md:38]:
```suggestion
Publishing models in 3ds Max is pretty straightforward. Create your model as you
```

File: website/docs/artist_hosts_3dsmax.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+need. You might need to adhere to specifications of your studio that can be different
+between studios and projects but by default your geometry does not need any
+other convention.
+
+![Model example](assets/blender-model_example.jpg)
```
Comment (BigRoy) [inline website/docs/artist_hosts_3dsmax.md:43]: Blender image?

File: website/docs/artist_hosts_3dsmax.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+### Creating instance
+
+Now create **Model instance** from it to let OpenPype know what in the scene you want to
+publish. Go **OpenPype → Create... → Model**.
+
+![Model create instance](assets/blender-model_create_instance.jpg)
+
+`Asset` field is a name of asset you are working on - it should be already filled
+with correct name as you've started Blender or switched context to specific asset. You
+can edit that field to change it to different asset (but that one must already exists).
```
Comment (BigRoy) [inline website/docs/artist_hosts_3dsmax.md:54]:
```suggestion
`Asset` field is a name of asset you are working on - it should be already filled
with correct name as you've started 3ds Max or switched context to specific asset. You
can edit that field to change it to different asset (but that one must already exists).
```

File: website/docs/artist_hosts_3dsmax.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+And that's it, you have your first model ready to publish.
+
+Now save your scene (if you didn't do it already). You will notice that path
+in Save dialog is already set to place where scenes related to modeling task on
+your asset should reside. As in our case we are working on asset called
+**character1** and on task **modeling**, path relative to your project directory will be
+`project_XY/assets/character1/work/modeling`. The default name for the file will
+be `project_XY_asset_task_version`, so in our case 
+`simonetest_character1_modeling_v001.blend`. Let's save it.
```
Comment (BigRoy) [inline website/docs/artist_hosts_3dsmax.md:88]: Blender filename.

File: website/docs/artist_hosts_3dsmax.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+
+![Model create instance](assets/blender-save_modelling_file.jpg)
```
Comment (BigRoy) [inline website/docs/artist_hosts_3dsmax.md:90]: Blender image

File: website/docs/artist_hosts_3dsmax.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+
+### Publishing models
+
+Now let's publish it. Go **OpenPype → Publish...**. You will be presented with following window:
+
+![Model publish](assets/blender-model_pre_publish.jpg)
```
Comment (BigRoy) [inline website/docs/artist_hosts_3dsmax.md:96]: Blender image

File: website/docs/artist_hosts_3dsmax.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+For more detail see [Publisher](artist_tools_publisher).
+
+Items in left column are instances you will be publishing. You can disable them
+by clicking on square next to them. White filled square indicate they are ready for
+publishing, red means something went wrong either during collection phase
+or publishing phase. Empty one with gray text is disabled.
+
+See that in this case we are publishing from the scene file
+`simonetest_character1_modeling_v001.blend` the Blender model named 
+`character1_modelDefault`.
```
Comment (BigRoy) [inline website/docs/artist_hosts_3dsmax.md:108]:
```suggestion
See that in this case we are publishing from the scene file
`simonetest_character1_modeling_v001.3ds` the 3ds Max model named 
`character1_modelDefault`.
```

**Review by BigRoy — DISMISSED** [review]
I think this still needs the changes from "blender" to "3ds Max"

File: website/docs/artist_hosts_3dsmax.md [hunk 8]
```diff
@@ -0,0 +1,226 @@
...
+In our case, it is `character1_rigDefault`. All the selected armature and models
+have been linked in this new collection. You should end up with something like
+this:
+
+![Blender - Rig Hierarchy Example](assets/blender-rig_hierarchy_before_publish.jpg)
+
+### Publishing rigs
+
+Publishing rig is done in same way as publishing everything else. Save your scene
+and go **OpenPype → Publish**. For more detail see [Publisher](artist_tools_publisher).
```
[the remark this answers is not in the document]
Comment (LiborBatek) [reply website/docs/artist_hosts_3dsmax.md:198]: @BigRoy sorry its a part of our internal training we are aware of the issues :)

Thx anyway...and sorry for bothering you!

**Review by jakubjezek001 — APPROVED** [review]

**Pull request merged**